packages/create-amplify/src/npm_package_manager_controller.ts

```diff
@@ -32,13 +32,9 @@ export class NpmPackageManagerController implements PackageManagerController {
     }
 
     try {
-      await executeWithLogger(this.execa, this.projectRoot, 'npm', args);
+      await executeWithDebugLogger(this.projectRoot, 'npm', args, this.execa);
     } catch {
-      throw new Error(
-        `\`npm install ${packageNames.join(' ')}${
-          typeDev ? ' --save-dev' : ''
-        }\` did not exit successfully.`
-      );
+      throw new Error(`\`npm ${args.join(' ')}\` did not exit successfully.`);
```

Comment:
couldn't we just propagate the error from `executeWithDebugLogger`? That already includes basically the same error message.

This applies to other places where we try/catch an `executeWithDebugLogger` statement